src/main/java/at/hannibal2/skyhanni/config/features/chat/BridgeFormatterConfig.java

```diff
@@ -0,0 +1,70 @@
+package at.hannibal2.skyhanni.config.features.chat;
+
+import at.hannibal2.skyhanni.config.FeatureToggle;
+import at.hannibal2.skyhanni.features.chat.BridgeFormatter;
+import com.google.gson.annotations.Expose;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorBoolean;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorButton;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorInfoText;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigEditorText;
+import io.github.notenoughupdates.moulconfig.annotations.ConfigOption;
+
+public class BridgeFormatterConfig {
+    @Expose
+    @ConfigOption(
+        name = "Enable Bridge Formatting",
+        desc = "Reformat guild bridge messages."
```

Comment:
Please be more descriptive (assuming someone doesn't know what a "bridge bot" is.)